about.html

```diff
@@ -0,0 +1,108 @@
+      <i class="closeIcon material-icons">close</i>
+    </button>
+  </nav>
+ </header>
+ <main class="main">
+  <section class="main-page">
+    <div class="main-div">
+      <p class="header-text">"Hello SCUTTERS!"</p>
+      <h1 class="program-text">Annual Alumni Meetup, Dhaka 2022</h1>
+      <p class="program-details">All the alumnies of "SCUT Alumni Association Bangladesh" from all over the world are welcome to meet the very first meetup in Dhaka. Our honrable alumnies and all the current students from Bangladesh are expected to present in that event to make it success. Please see the detais on the about section and order a ticket to reserve your seat.</p>
+      <p class="festivalContact">Please contact us per Email for any further queestions
+         <br />about SCUT Alumni Meetup! <br /><a href="toemail:masuduzzaman_811@outlook.com">
+           scut_meet@contactme.com</a></p>
+    </div>
+  </section>
+  <section class="mainSection">
+    <h2 class="mainHeader">SCUT Alumni Meetup 2020</h2>
+    <hr />
+    <p class="mainText">Lorem ipsum dolor, sit amet consectetur adipisicing elit.
+       Neque, quas obcaecati! Error vero doloribus architecto quam commodi, sequi
+        culpa quasi consequuntur ea, suscipit, voluptates optio eius sunt nostrum
+         itaque officiis.</p>
+    <img class="mainLogo" src="image/icons/capstone1.png" alt="logo" >
+  </section>
+  <section class="pastFestivals">
+    <hr class="hLine" />
+    <h2 class="past-events-header">See the past SCUT Alumni Association Bangladesh Events</h2>
+    <hr />
+    <p class="past-events-description">Take a look at our last two events which took place in Cox's Bazar and The Palace, Sylhet. </p>
+    <div class="event-container">
+      <article class="card cardOne">
+        <div>
+          <h3>2018</h3>
+          <p>Cox's Bazar Internation Stadium, COX'S BAZAR</p>
+        </div>
+      </article>
+      <article class="card cardTwo">
+        <div>
+          <h3>2020</h3>
+          <p>The Palace Resort, Sylhet</p>
+        </div>
+      </article>
+    </div>
+  </section>
+</main>
+<footer id="footer">
+  <section class="sponsor">
+    <h2 class="sponsor-head">Sponsors</h2>
+    <hr>
+    <div class="sponsor-images-container">
+      <img class="sponsor-img" src="image/images/partners/airbnb.png" alt="sponsor-img">
+      <img class="sponsor-img" src="image/images/partners/navar.png" alt="sponsor-img">
+      <img class="sponsor-img" src="image/images/partners/daumk.svg" alt="sponsor-img">
+      <img class="sponsor-img" src="image/images/partners/Google.png" alt="sponsor-img">
+      <img class="sponsor-img" src="image/images/partners/mozilla.png" alt="sponsor-img">
+    </div>
+  </section>
+  <section class="footer">
+    <img class="footerLogo" src="image/icons/capstone1.png" alt="logo" />
+    <p class="footerText">
+      2022, SCUT Alumni Association Bangladesh, Some rights reserved.</p>
+  </section>
+</footer>
+<script src="script.js"></script>
+</body>
```

Comment:
- [ ] Please, consider decreasing the partners icons size - it is too big for this section
- [ ] Please, consider giving some padding to footer, make it closer to original design 👍 

> Your version
<img width="311" alt="Screen Shot 2022-04-22 at 10 10 51" src="https://user-images.githubusercontent.com/49064106/164614384-223da64f-46b1-4009-bc02-8c960e991cbc.png">

> Original version
<img width="492" alt="Screen Shot 2022-04-22 at 10 11 33" src="https://user-images.githubusercontent.com/49064106/164614396-05cc28cb-e3d0-4633-8e2a-f50d9ce29cdf.png">

Reply:
1. Partner logo sizes have been decreased for medium and small screen sizes.
2. Padding has been added in the footer section to make it looks closer to the original design.

Reply:
resolved 👍